src/renderer/store/index.js

```diff
@@ -16,3 +18,13 @@ export default new Vuex.Store({
   ],
   strict: process.env.NODE_ENV !== 'production',
 });
+
+const previousMigration = store.state.Migrations.last;
```

Comment:
I changed the `importer` schema, to contain an `id` property.
So now, when the user updates his program, the `Importers.vue` expect for `importer.id`, but the user hasn't this property because the `importer` is there from a previous version.

Since we using the state directly (with `mapState`) and not expose all the data with getters (see #62 ) we can't control that each importer has an `id`, so we need to run a script to update the `store` at the loading time.

The store will remember which script ran.